### ISSUE TITLE
Medically controlled hypertension

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7409]
**Preferred term label:**
Medically controlled hypertension

**Synonyms**

**Definition (free text, please give PubMed ID)**
The presence of chronic increased pressure in the systemic arterial system but below 140/90 mm Hg or, if the subject had diabetes or chronic respiratory disease, below 130/80 mm Hg (PMID: 19602664).

**Parent term (use hpo.jax.org/app)**
Increased blood pressure HP:0032263

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by jagcsun** [comment]
Prehypertension (blood pressure between 120-139/80-89 mmHg (PMID: 17893682)) also sounds same/similar as the definition of Medically controlled hypertension. But not sure if both terms are different or same since Medically controlled hypertension definition accommodates different range for people with some conditions. 

@pnrobinson

**Comment by PaulNSchofield** [comment]
Hypertension is normally graded: Normal - SBP 120-129 mmHg and/or DBP 80-84 mmHg.
High normal - SBP 130-139 mmHg and/or DBP 85-89 mmHg.
Grade 1 - SBP 140-159 mmHg and/or DBP 90-99 mmHg.
Grade 2 - SBP 160-179 mmHg and/or DBP 100-109 mmHg.
Ive not come across pre-hypertension used in a generally accepted way. Clinically controlled hypertension refers to the effect of medication/therapy and can reflect etiology or personalised response to drugs but I would not define it in the way suggested. It would on the face of it be described as Normal or high normal range hypertension in a person previously diagnosed as hypertensive under therapy. Generally the category of "uncontrolled hypertension" is used and control is taken as the default. Controlled hypertension also tends to be a medium term diagnosis. Not sure that this is a useful  concept therefore.

**Comment by pnrobinson** [comment]
I think that we should represent treatments with phenopackets. This is a slightly difficult concept but I do not think it would be good for HPO right now. Thanks @PaulNSchofield
